astrokat/test/test_nd_timings.py

```diff
@@ -1,17 +1,21 @@
+###############################################################################
+# SKA South Africa (http://ska.ac.za/)                                        #
+# Author: cam@ska.ac.za                                                       #
+# Copyright @ 2019 SKA SA. All rights reserved.                               #
+#                                                                             #
+# THIS SOFTWARE MAY NOT BE COPIED OR DISTRIBUTED IN ANY FORM WITHOUT THE      #
+# WRITTEN PERMISSION OF SKA SA.                                               #
+###############################################################################
+
+from __future__ import absolute_import
 from __future__ import print_function
 
-import logging
 import os
-import sys
-
 import unittest2 as unittest
-from astrokat import observe_main, simulate
-
-if sys.version_info[0] == 3:
-    from io import StringIO
-else:
-    from StringIO import StringIO
+from mock import patch
 
+from astrokat import observe_main
+from .astrokat_testutils import yaml_path, LoggedTelescope
```

Comment:
Suggest module name is `testutils` or `test_utils`, rather than `astrokat_testutils`.  No need to mention astrokat inside astrokat.